.github/workflows/deploy-local-acs.yml

```diff
@@ -0,0 +1,168 @@
+      #       echo "Enabling clustered tests with 2 replicas"
+      #       echo "helm_install_params=--set alfresco-repository.replicaCount=2" >> "$GITHUB_OUTPUT"
+      #     fi
+
+      - name: Add dependency chart repos
+        run: |
+          helm repo add self https://alfresco.github.io/alfresco-helm-charts/
+          helm repo add elastic https://helm.elastic.co/
+
+      # Name and value to be fetched by automation??
+      - name: Helm install
+        run: >-
+          helm dep build ./helm/alfresco-content-services &&
+          helm install acs ./helm/alfresco-content-services
+          --set global.search.sharedSecret="$(openssl rand -hex 24)"
+          --set global.known_urls=http://localhost
+          --set global.alfrescoRegistryPullSecrets=regcred
+          --values helm/${{ matrix.name }}/${{ matrix.values }}
+          --values test/enterprise-integration-test-values.yaml
+
+      # what about values here??
+      - name: Watch Helm deployment
+        env:
+          VALUES_FILE: ${{ matrix.values }}
+        run: |
+          kubectl get pods --watch &
+          KWPID=$!
+          kubectl wait --timeout=7m --all=true --for=condition=Available deploy && kill $KWPID
+          if [ "${VALUES_FILE:0:2}" != "7." ] ; then echo -n "Waiting for ESC Reindexing job to complete... "
+            kubectl wait --timeout=5m --for=condition=complete job/acs-alfresco-search-enterprise-reindexing
+            echo "Completed."
+          fi
+
+      - name: Spit cluster status after install
+        if: always()
+        run: |
+          helm ls --all-namespaces --all
+          helm status acs --show-resources
+          kubectl describe pod
+
+      - name: Run Newman tests
+        uses: nick-fields/retry@7152eba30c6575329ac0576536151aca5a72780e # v3.0.0
+        id: newman
+        with:
+          timeout_minutes: 1
+          retry_wait_seconds: 20
+          max_attempts: 5
+          command: >-
+            docker run --network=host
+            -v $(pwd)/test/postman:/etc/postman
+            -t postman/newman run /etc/postman/helm/acs-test-helm-collection.json
+            --global-var protocol=http --global-var url=localhost
+
+      - name: Spit cluster status after newman tests
+        if: always() && steps.newman.outcome != 'skipped'
+        run: |
+          kubectl get all --all-namespaces
+          kubectl describe pod
+          kubectl events --for deployment/acs-alfresco-repository
+
+      - name: Run helm test
+        id: helm_test
+        run: helm test acs
+
+      - name: Spit cluster status after helm test
+        if: always() && steps.helm_test.outcome != 'skipped'
+        run: |
+          kubectl logs -l app.kubernetes.io/component=dtas --tail=-1
+          kubectl get all --all-namespaces
+          kubectl describe pod
+          kubectl events --for deployment/acs-alfresco-repository
```

Comment:
```suggestion
```

this is our e2e, no need to run them here as well

Reply:
Removed